common/styles/components/_work_media.scss

```diff
@@ -7,6 +7,17 @@
     height: calc(100vh - 200px);
   }
 
+  // TODO: tidy this up if/when it becomes a reality
```

Comment:
I've gone for labelling my todos to find later e.g. `TODO: (work redesign) ...` if that's useful?